runtime/CSharp/Antlr4BuildTasks/Antlr4ClassGenerationTaskInternal.cs

```diff
@@ -244,8 +244,9 @@ public bool Execute()
                 }
 
                 arguments.Add("-o");
-                arguments.Add(OutputPath);
-
+                // Wrap output path with quotes to work with cmdline
+                arguments.Add("\""+ OutputPath + "\"");
```

Comment:
This is obviously not right, as ```JoinArguments``` is supposed to append quotes to arguments with spaces.